test/data/testAppObj.js

```diff
@@ -5,12 +5,7 @@ define(['./jobsData'], (JobsData) => {
     return {
         app: {
             fileParamIds: {
-                fastq_reads: [
-                    'fastq_fwd_staging_file_name',
-                    'fastq_rev_staging_file_name',
-                    'sra_staging_file_name',
-                    'name',
-                ],
+                fastq_reads: ['fastq_fwd_staging_file_name', 'fastq_rev_staging_file_name', 'name'],
```

Comment:
`sra_staging_file_name` isn't actually a parameter in the faked out data, and that was causing problems.